config/routes.rb

```diff
@@ -52,6 +52,9 @@
 
     namespace :v1 do
       concern :participant_actions, Participants::Routing.new
+
+      resources :npq_funding, only: [:index], path: "npq-funding"
```

Comment:
Is this more of a show? We're looking up data on a single TRN and getting one hash of data back.

Reply:
sure, I've changed this to a `show` now